src/main/java/com/olgaboiar/mint/Logger.java

```diff
@@ -0,0 +1,23 @@
+package com.olgaboiar.mint;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class Logger {
+
+    public void logToConsole(String message) {
```

Comment:
Can you combine `logToConsole` and `logToFile` into one method, and allow user to set the destination or strategy separately? 😄

Reply:
refactored in https://github.com/olgaboiar/MintHTTPServer/pull/6/commits/0927485dc63e8abe53c505c389574ea7ede310fe